google-cloud-bigtable/test/helper.rb

```diff
@@ -259,3 +259,8 @@ def read_rows_acceptance_test_data
 
   tests
 end
+
+# A microseconds integer rounded to the nearest millisecond. For example: `1564257960168000`.
+def timestamp_micros
```

Comment:
I think this method should be in the helper class, not on main.

Reply:
Thanks, moved.